accepted/2020/net5/net5.md

```diff
@@ -0,0 +1,503 @@
+make it easier to use, she's adding a new `SkiaSharpImage` type, which
+represents a bitmap and is constructed via OS-provided data types. Ada uses
+`#if` to expose different constructors on different platforms:
+
+```C#
+public static class SkiaSharpImage
+{
+#if ANDROID
+    public SkiaSharpImage(Android.Media.Image nativeImage) { /* ... */  }
+#endif
+
+#if IOS
+    public SkiaSharpImage(NSImage nativeImage) { /* ... */ }
+#endif
+
+#if WINDOWS
+    public SkiaSharpImage(Windows.Media.BitmapImage nativeImage) { /* ... */ }
+#endif
+}
+```
+
+## Requirements
+
+### Goals
+
+* Use naming that aligns with product strategy
+* Merge .NET Core and .NET Standard into a single concept
+* Developers should be able to understand compatibility relationships without
+  having to consult a mapping table.
+* Provide compatibility with existing concepts and NuGet packages
+* It would be great to get this into .NET 5 Preview 1
+
+### Non-Goals
+
+* Replace TFMs or expand runtime identifiers (RIDs)
+* Support multi-targeting between OS versions
+
+## Design
+
+| TFM             | Compatible With                                            | Comments                          |
+|-----------------|------------------------------------------------------------|-----------------------------------|
+| net5.0          | net1..4 (with NU1701 warning)                              | No WinForms or WPF                |
+|                 | netcoreapp1..3.1 (warning when WinForms/WPF is referenced) |                                   |
+|                 | netstandard1..2.1                                          |                                   |
+| net5.0-android  | xamarin.android                                            |                                   |
+|                 | (+everything else inherited from net5.0)                   |                                   |
+| net5.0-ios      | xamarin.ios                                                |                                   |
+|                 | (+everything else inherited from net5.0)                   |                                   |
+| net5.0-macos    | xamarin.macos                                              |                                   |
+|                 | (+everything else inherited from net5.0)                   |                                   |
+| net5.0-tvos     | xamarin.tvos                                               |                                   |
+|                 | (+everything else inherited from net5.0)                   |                                   |
+| net5.0-watchos  | xamarin.watchos                                            |                                   |
+|                 | (+everything else inherited from net5.0)                   |                                   |
+| net5.0-win      | netcoreapp1..3.1                                           | WinForms + WPF                    |
```

Comment:
i'd imagine this should be "net5.0-windows". this sticks out as the only abbreviation...and it should be the brand we want to treat as nicely as the others.

Reply:
That's a good point. I've modelled it off of the RID we currently use. I can go either way. @ericstj, what are your thoughts on this?

Reply:
I'd prefer win, for the same argument: It's already used in placed like RID, and we all know what it means.

Reply:
RID is much less common or visible than TFM. i'm not worried about confusion. Just don't know why we'd abbreviate windows, but not android, macos, etc...

Reply:
I've changed it to `windows`.